subprojects/docs/src/docs/userguide/running-builds/build_environment.adoc

```diff
@@ -134,6 +134,12 @@ When set to `all` or `full`, a stacktrace is present in the output for all excep
 Using `full` doesn't truncate the stacktrace, which leads to a much more verbose output.
 _Default is `internal`._
 
+`org.gradle.welcome=(never,once)::
```

Comment:
There is a missing backtick after the `)` on this line.  You can see that the property is not formatted correctly on the [doc page](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties).  Also it is out of alphabetical order.  It should go after `org.gradle.warning.mode`.  `org.gradle.logging.stacktrace` is also in the wrong location - it should go after `org.gradle.logging.level`

Reply:
Submitted #22702 and PR#22703